### PR TITLE
Initial Product CRUD

### PR DESCRIPTION
**Pull request by anfimovdm**

### CODE REVIEW

File: alws/routers/docs.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+from fastapi import APIRouter, status
+from fastapi.responses import JSONResponse, HTMLResponse
+import markdown
+
+from alws.config import settings
+
+
+public_router = APIRouter(
+    prefix='/docs',
```
Comment (Korulag) [inline alws/routers/docs.py:12]: This will override standard docs from FastAPI

**Review by Korulag — CHANGES_REQUESTED** [review]

File: alws/crud/build.py [hunk 2]
```diff
@@ -196,8 +197,17 @@ async def remove_build_job(db: Session, build_id: int) -> bool:
...
+                product.name
+                for product in build.products
+            ))
+            raise BuildError(
+                f"Cannot delete Build={build_id}, "
+                f"build contains in following products:\n{product_names}"
+            )
         if build.released:
-            return False
+            raise BuildError(f"Build with {build_id=} is released")
```
Comment (javihernandez) [inline alws/crud/build.py:210]: Extra **=** character

**Review by javihernandez — APPROVED** [review]
Reviewed the code and couldn't find anything relevant to comment on aside from what I already commented to @anfimovdm in private. I also tested the code in staging following the test plan and spotted a few minor issues that I documented [here](https://docs.google.com/document/d/11TK5UUOpDwJM0Ei-UifmOoCYzF6WBy0iBI1UalBUH5A/edit?usp=sharing). Overall, it looks good to me.

**Review by soksanichenko — APPROVED** [review]
lgtm

**Review by maccelf — APPROVED** [review]

File: alws/crud/build.py [hunk 3]
```diff
@@ -205,8 +206,17 @@ async def remove_build_job(db: Session, build_id: int) -> bool:
...
```
Comment (Korulag) [inline alws/crud/build.py:219]: doesn't the `=` sign cause troubles when being in this code branch?

**Review by Korulag — APPROVED** [review]

File: alws/crud/build.py (hunk 3 above)
Comment (anfimovdm) [reply alws/crud/build.py:219]: I'm not sure, but I'll check it

Comment (anfimovdm) [reply alws/crud/build.py:219]: image
works fine

**Review by Korulag — APPROVED** [review]

**Pull request merged**